app/src/main/java/org/mozilla/tv/firefox/webrender/FirefoxProgressBar.kt

```diff
@@ -40,7 +31,7 @@ class FirefoxProgressBar @JvmOverloads constructor(
         if (loading) {
             showBar()
         } else {
-            scheduleHideBar()
+            hideBar()
```

Comment:
I wonder if there are pages where it loads super quickly so you don't even see the progressbar? But that's probably fine.